Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -245,7 +242,7 @@ class TabTrayViewController: UIViewController {
 
         navigationMenu.snp.makeConstraints { make in
             make.width.lessThanOrEqualTo(343)
-            make.height.equalTo(ChronologicalTabsControllerUX.navigationMenuHeight)
+            make.height.equalTo(32)
```

Comment:
This was a constant and not anymore? Could it be a UX constant again (and I guess for the 343 as well)?